packages/std/src/ibc.rs

```diff
@@ -438,10 +438,7 @@ impl IbcPacketTimeoutMsg {
 /// will use other Response types
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[non_exhaustive]
-pub struct IbcBasicResponse<T = Empty>
-where
-    T: CustomMsg,
-{
+pub struct IbcBasicResponse<T = Empty> {
```

Comment:
Very good to remove struct trait restrictions. This will become annoying if we multitest ibc contracts